lib/pure/hashes.nim

```diff
@@ -121,16 +121,32 @@ proc hash*(x: string): THash =
     h = h !& ord(x[i])
   result = !$h
 
+# The following need a better home than "hashes":
+const magicIdentSeparatorRuneByteWidth* = 3
+
+proc isMagicIdentSeparatorRune*(cs: cstring, i: int): bool  {. inline } =
+  result =  ord(cs[i]) == 226 and 
+            ord(cs[i + 1]) == 128 and
+            ord(cs[i + 2]) == 147     # en-dash  # 145 = nb-hyphen
+
 proc hashIgnoreStyle*(x: string): THash =
```

Comment:
move your new hashIgnoreStyle to idents.nim then. I think.actually I'm not sure yet.